apps/web/playwright/auth/forgot-password.e2e.ts

```diff
@@ -16,6 +16,8 @@ test.describe("Forgot password", async () => {
     await page.goto("/auth/forgot-password");
     await page.waitForSelector("text=Forgot Password?");
 
+    // eslint-disable-next-line playwright/no-wait-for-timeout
+    await page.waitForTimeout(300);
```

Comment:
Because page loads are faster now, fill event fails irregularly, making tests flaky

Reply:
This is an anti-pattern for a reason. Let's rethink our approach here. We might wait for a specific endpoint response or another selector.

Reply:
true, hmm what could it be

can you use the app locally and see if you have a clue?